libvast/src/system/passive_partition.cpp

```diff
@@ -196,9 +196,20 @@ partition_actor::behavior_type passive_partition(
   self->state.filesystem = std::move(filesystem);
   self->state.name = "partition-" + id_string;
   VAST_TRACEPOINT(passive_partition_spawned, id_string.c_str());
+  self->set_down_handler([=](const caf::down_msg& msg) {
+    if (msg.source != self->state.store.address()) {
+      VAST_WARN("{} ignores DOWN from unexpected sender: {}", *self,
+                msg.reason);
+      return;
+    }
+    VAST_ERROR("{} cannot reach its {} store and shuts down: {}", *self,
+               self->state.store_id, msg.reason);
```

Comment:
The grammar sounds a bit off, I suggest `{} shuts down after DOWN from store {}: {}"`.